_docs/aws_resources.md

```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
    * [Delete S3 buckets](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/delete-bucket.html)
    * [Terminate EC2 instances -- leaving EC2 instances running is a common cause of unexpected recurring charges](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html)
    * Delete database tables: [DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-8.html), [RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_DeleteInstance.html), and/or [Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-clusters-console.html#delete-cluster)
-  
-The list above should be sufficient to get you started. You **do not** have to follow every step. However, pay close attention to **1, 3, and 5** because those are some of the most common solutions. 
 
-For additional information, AWS has a [comprehensive guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html) to help you avoid unexpected charges if you find yourself in an uncharted territory. 
+The list above should be sufficient to get you started. You **do not** have to follow every step. However, pay close attention to **1, 3, and 5** because those are some of the most common solutions.
 
+For additional information, AWS has a [comprehensive guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html) to help you avoid unexpected charges if you find yourself in an uncharted territory.
+
+
+
+# Video Tutorials To Help You Get Familiar With AWS Cloud services
+
+## Static Website Hosting on S3, Route 53 Domain Name Registration
+In this AWS S3 tutorial the presenter walks step-by-step through the process of hosting a website on Amazon S3. Then set up a custom domain name in AWS Route 53 and associate it with our website in S3
+(Domain registration is not free but at an affordable price. )
+
+[S3 Video Tutorial Available Here ] (https://www.youtube.com/watch?v=P_B79xXf-w8)
```

Comment:
Link syntax is very, very particular; can't have spaces between the `]` bracket and `(` paren. Take a look at the [rendered markdown](https://github.com/UWB-ACM/Hackathon-Docs-2020/blob/490cd5ec451855cfa37d06f50d5bf9cb4161e5c3/_docs/aws_resources.md) to see what I'm talking about.

```suggestion
[S3 Video Tutorial Available Here](https://www.youtube.com/watch?v=P_B79xXf-w8)
```